### DISCUSSION TITLE
Podman 5 for RHEL

### DISCUSSION THREAD

**Discussion by zackman0010** [containers/podman#22592] [Q&A] [answered]
Does anyone know when Podman 5 will be hitting the RHEL repositories? And if it does, will it be making it to RHEL 8 or will it only be available on RHEL 9?

**Comment by Luap99** [comment, marked as answer]
it will not be shipped on RHEL 8, for RHEL 9 the target is 9.5

**Reply by zackman0010** [reply]
Thanks!

**Comment by putradimas** [comment]
Is there any way to install podman 5 in rhel8? Like a yum repo?
